Lake/Load/Materialize.lean

```diff
@@ -6,6 +6,7 @@ Authors: Gabriel Ebner, Sebastian Ullrich, Mac Malone
 import Lake.Util.Git
 import Lake.Load.Manifest
 import Lake.Config.Dependency
+import Lake.Config.Package
```

Comment:
This is a very expensive import. I think `materializePackageEntry` can do without it (as `materializeDep` did before).

Reply:
Does it matter?  `Lake.Config.Package` gets imported by lakefiles anyhow.

Reply:
Yeah, it generally makes rebuilding faster when developing, as changes in to Package don't effect this low-level manifest code and vice versa.

Reply:
Okay, removed the import at the expense of having a couple of stringly-typed `FilePath` arguments.